src/main/steps/respondent/respondentcaseSequence.ts

```diff
@@ -81,17 +78,6 @@ export const respondentCaseSequence: Step[] = [
     showInSection: Sections.AboutRespondentCase,
     getNextStep: () => RESPONDENT_TASK_LIST_URL,
   },
-
-  {
```

Comment:
Where are these steps moved now?

Reply:
These are part of response case sequence now. These occured due to duplication.